### ISSUE TITLE
Filtering by comparing two columns

### ISSUE THREAD

**Issue by goksel22** [bhrnjica/daany#12]
Hello,
Daany was a feature that I have been looking for a long time and was missing for me in .net. It provided me great convenience with easy and fast field and data addition. Can I do filtering by comparing the columns I use in python dataframe in daany?
Example
df[df["a"]>df["b"]]

**Comment by bhrnjica** [comment]
Yes sure you can do that, first by creating calculated column and then use filter feature. 
Please refer to DevGuid for the example of calculated column and filtering.
